here-naksha-lib-psql/src/commonMain/kotlin/naksha/psql/executors/query/WhereClauseBuilder.kt

```diff
@@ -115,6 +120,16 @@ class WhereClauseBuilder(private val request: ReadFeatures) {
         }
     }
 
+    private fun resolveTransformation(transformation: SpTransformation, geometryStmt: String): String {
+        return when(transformation){
+            is SpBuffer -> "ST_Buffer($geometryStmt, ${transformation.distance})"
```

Comment:
dinstance parameter should be passed as argument, it will protect us in the future from SQL injection (at the moment it's safe because it's a double, but if someone change it to string this code won't be safe anymore)

Reply:
good one, will apply

Reply:
done